addons/l10n_es_edi_facturae/data/facturae_templates.xml

```diff
@@ -0,0 +1,460 @@
+        </template>
+
+        <!-- Sub-template used for every instance of AccountType -->
+        <template id="account_type">
+            <IBAN t-out="account['IBAN']"/>
+            <AccountNumber t-out="account['AccountNumber']"/>
+            <BankCode t-out="account.get('BankCode')"/>
+            <BranchCode t-out="account.get('BranchCode')"/>
+            <t t-call="l10n_es_edi_facturae.address_type"><t t-set="address"  t-value="account['Address']"/></t>
+            <BIC t-out="account.get('BIC')"/>
+            <CollectionAdditionalInformation t-out="account.get('CollectionAdditionalInformation')"/>
+            <RegulatoryReportingData t-out="account.get('RegulatoryReportingData')"/>
+            <DebitReconciliationReference t-out="account.get('DebitReconciliationReference')"/>
+            <FactoringAssignmentClauses t-out="account['FactoringAssignmentClauses']"/>
+        </template>
+
+        <!-- Sub-template used for every instance of InstallmentType -->
+        <template id="installment_type">
+            <Installment>
+                <InstallmentDueDate t-out="installment['InstallmentDueDate']"/>
+                <InstallmentAmount t-out="installment['InstallmentAmount']"/>
+                <PaymentMeans t-out="installment['PaymentMeans']"/>
+                <AccountToBeCredited t-if="installment.get('AccountToBeCredited')">
+                    <t t-call="l10n_es_edi_facturae.account_type"><t t-set="account" t-value="installment['AccountToBeCredited']"/></t>
+                </AccountToBeCredited>
+                <PaymentReconciliationReference t-out="installment.get('PaymentReconciliationReference')"/>
+                <AccountToBeDebited t-if="installment.get('AccountToBeDebited')">
+                    <t t-call="l10n_es_edi_facturae.account_type"><t t-set="account" t-value="installment['AccountToBeDebited']"/></t>
+                </AccountToBeDebited>
+                <CollectionAdditionalInformation t-out="installment.get('CollectionAdditionalInformation')"/>
+                <RegulatoryReportingData t-out="installment.get('RegulatoryReportingData')"/>
+                <RegulatoryReportingData t-out="installment.get('DebitReconciliationReference')"/>
+            </Installment>
+        </template>
+
+        <!-- Sub-template used for every instance of LegalEntityType -->
+        <template id="legal_entity_type">
+            <LegalEntity>
+                <CorporateName t-out="legal_entity['CorporateName']"/>
+                <TradeName t-out="legal_entity.get('TradeName')"/>
+                <RegistrationData t-if="legal_entity.get('RegistrationData')">
+                    <Book t-out="legal_entity['RegistrationData'].get('Book')"/>
+                    <RegisterOfCompaniesLocation t-out="legal_entity['RegistrationData'].get('RegisterOfCompaniesLocation')"/>
+                    <Sheet t-out="legal_entity['RegistrationData'].get('Sheet')"/>
```

Comment:
First you create entire dictionaries, then you pass them to the QWeb, so if you want to know what is in the qweb you need to check one step further.